src/common/config/index.js

```diff
@@ -324,6 +327,24 @@ export default class Config extends EventEmitter {
     return newTeams;
   }
 
+  /**
+   * Apply a default sort order to the team list, if no order is specified.
+   * @param {array} teams to sort
+   */
+  sortUnorderedTeams(teams) {
+    let newTeams = teams;
+    let i = 0;
+
+    newTeams = newTeams.filter((newTeam) => {
+      if (!newTeam.order || newTeam.order < 0) {
+        newTeam.order = i++;
+      }
+
+      return newTeam; // eslint-disable-line max-nested-callbacks
+    });
+    return newTeams;
+  }
+
```

Comment:
I think we can refine this algorithm, people can add random order (not negative but I can setup 281 as order on a two server setup), so anything depending on user input needs to be resilient.  With that in mind I have a few questions:

what would happen if I had 5 servers and the last one has the order value of 3?

I'm not convinced about using filter here, the only way to filter is if newteam is falsy (null, undefined in this case) and that shouldn't happen, but in case it does it would fail on line 339. How about transition it to either map or reduce?

do we mind having gaps in the order (e.g.: user added an order value of 5 in a 3 server setup)? My guess is that it is going to be fine, but we would have to double check any code depending on order value is expecting gaps. Another option would be to remove the gaps if they exist.

Reply:
I based this code off of the surrounding functions, but am open to other ideas.  You are correct that users that specify nonsequential order flags may clash with this; If the config file specifies order:1 on one entry and leaves two others blank, you would end up with [0, 1, 1].

If order is merely consumed as a sort indicator, this might be "close enough" to what users expect to see; If it needs to be unique (I need to dig through the code and see), this could lead to a crash.

Reply:
I've pushed a change to the code that I think addresses this.  It makes two passes on the user's defaultTeams object, first to sort the list numerically by [order], and then again to renumber the entire list from 0..max.  This ensures the user's specified sort order is preserved, while also making sure that we have an item with order:0, which seems to be required for the first tab to be auto-raised on startup.

(There may be another way to fix this, but I'm sticking with the only section of code I've figured out so far.  :) )